packages/xo-server/src/xo-mixins/backups-ng/index.js

```diff
@@ -650,7 +651,7 @@ export default class BackupNg {
             event: 'task.info',
             taskId: runJobId,
             data: {
-              vms: vms.map(vm => vm.id),
+              vms: map(vms, 'id'),
```

Comment:
Why?

Reply:
'vms' can be an object is due to the result of `value-matcher`.

Reply:
We should make sure it's always a dictionary and then use `Object.keys()`.